sound/soc/sof/topology.c

```diff
@@ -2150,12 +2144,29 @@ static int sof_complete(struct snd_soc_component *scomp)
 					if (ret < 0)
 						return ret;
 				}
+
```

Comment:
in the loop above, should we skip the case where swidget->id == comp_swidget->id?

It's a bit confusing when some widgets belong to another type of widget....

Reply:
I think this issue got fixed in the latest rebase.

Reply:
not following @plbossart . the first loop is for the pipeline_list and it contains only scheduler widgets

Reply:
I don't understand what GitHub is doing, the comment was about a double loop which has now vanished.